src/basic/forward.h

```diff
@@ -310,6 +310,11 @@ typedef struct VeritySettings VeritySettings;
 #define AT_SYMLINK_FOLLOW       0x400
 #define AT_SYMLINK_NOFOLLOW     0x100
 
+#define R_OK 4
```

Comment:
Ugh, forward.h is supposed to be about headers only. If a .c file needs these macros, they should always get proper definition from standard libc headers. I'm wary of this change.

Reply:
That opposes to the concept of forward.h. We already have AT_ macros here. Why not to add the three? What's the difference between them?

Reply:
But in this case, the opinion is not strong compared to the other discussed points. Will change.

Reply:
Dropped the change, and now fcntl.h is included by fs-util.h.